src/components/network/network-table.js

```diff
@@ -713,22 +741,49 @@ const NetworkTable = (props) => {
             }
 
             if (isLineOnEditMode(rowData)) {
-                return (
-                    <div key={key} style={style}>
-                        <div className={classes.editCell}>
-                            {lineEdit.errors.size === 0 && (
+                if (rowIndex === 1) {
+                    // The current line is in edit mode and is the top one.
+                    return (
+                        <div
+                            key={key}
+                            style={style}
+                            className={clsx(
+                                classes.topEditRow,
+                                classes.leftFade
+                            )}
+                        >
+                            <div className={classes.editCell}>
+                                {lineEdit.errors.size === 0 && (
+                                    <IconButton
+                                        size={'small'}
+                                        onClick={() => commitChanges(rowData)}
+                                    >
+                                        <CheckIcon />
+                                    </IconButton>
+                                )}
                                 <IconButton
                                     size={'small'}
-                                    onClick={() => commitChanges(rowData)}
+                                    onClick={() => resetChanges(rowData)}
                                 >
-                                    <CheckIcon />
+                                    <ClearIcon />
                                 </IconButton>
-                            )}
-                            <IconButton
-                                size={'small'}
-                                onClick={() => resetChanges(rowData)}
-                            >
-                                <ClearIcon />
+                            </div>
+                        </div>
+                    );
+                }
+                // The current line is in edit mode, but is not the top one.
+                return (
+                    <div
+                        key={key}
+                        style={style}
+                        className={clsx(
+                            classes.referenceEditRow,
+                            classes.leftFade
+                        )}
+                    >
+                        <div className={classes.editCell}>
+                            <IconButton size={'small'}>
+                                <MoreHorizIcon />
                             </IconButton>
```

Comment:
Since it's a button but there's no event tied to it, it causes a bit of friction when hovering then clicking and nothing happens
Maybe it could be fixed by centering MoreHorizIcon without using IconButton or by disabling the change of background color on hover ?

Reply:
Ok, fixed it.